tests/cpu_bugs/ipmi_to_qemu.pm

```diff
@@ -35,23 +35,33 @@ sub run {
     my $current_dist;
     my $sles_running_version;
     my $sles_running_sp;
+    my $sles_running_spcurrent;
+    my $sles_running_versioncurrent;
+    my $current_distnow;
     script_run("systemctl disable apparmor.service");
     script_run("aa-teardown");
     if (get_var("DIST_FOR_REPO")) {
         ($sles_running_version, $sles_running_sp) = split(/sp/i, get_var("DIST_FOR_REPO"));
     } else {
-        ($sles_running_version, $sles_running_sp) = get_os_release();
+        die "Please check the DIST_FOR_REPO";
     }
+    ($sles_running_versioncurrent, $sles_running_spcurrent) = get_os_release();
     if ($sles_running_sp eq '0') {
-        $current_dist = sprintf("SLE_%s", $sles_running_version);
+        $current_dist = sprintf("%s.0", $sles_running_version);
     } else {
-        $current_dist = sprintf("SLE_%s_SP%s", $sles_running_version, $sles_running_sp);
+        $current_dist = sprintf("%s.%s", $sles_running_version, $sles_running_sp);
```

Comment:
Sorry that I'm not familiar with openqa testing process. Here I have a dummy question:
Since previously it's SLE_%s. then this change bring an thinking to me: now the name convention is this change, how about old product? like SLE15SP4? do we need consider backword compatibility? 
Or this is a invalid thinking?

Reply:
Thanks. Now the principle have changed eg:http://download.opensuse.org/repositories/devel:/openQA/
If 15SP3 maybe not adapt.